app/policies/wiki_policy.rb

```diff
@@ -0,0 +1,7 @@
+class WikiPolicy < ApplicationPolicy
+  class Scope < Scope
```

Comment:
You probably don't need this method at the moment since Scope will be inherited from the ApplicationPolicy, but it's not hurting anything.